dev/docs/builtins-reference.md

```diff
@@ -351,3 +353,32 @@ y = X %*% rand(rows = ncol(X), cols = 1)
 w = lm(X = X, y = y)
 ress = slicefinder(X = X,W = w, Y = y,  k = 5, paq = 1, S = 2);
 ```
+## `normalize`-Function
+
+The `normalize`-function is used for data preparation by normalization. Here, the values of a matrix (numeric columns) are changed in the dataset to use a common scale, while preserving differences in the ranges of values.
```

Comment:
The lines can be wrapped for readability. :)

358 .... (numeric columns)
359 are changed ...